src/util/transactionUtils.ts

```diff
@@ -14,8 +14,39 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
-import { TransactionStatusType, TransactionStatusEnum } from '../types/transactionTypes';
+import {
+  TransactionStatusType,
+  TransactionStatusEnum,
+  Payload,
+  TransactionDisplayPayload
+} from '../types/transactionTypes';
 
 export function isTransactionCompleted(transaction: TransactionStatusType): boolean {
   return transaction.status === TransactionStatusEnum.COMPLETED;
 }
+
+interface ToObject {
+  [propName: number]: number;
+}
```

Comment:
Why is this needed at all?

Reply:
it is not needed anymore after your proposal.